mgmt/rest/client/plugin.go

```diff
@@ -78,6 +79,63 @@ func (c *Client) UnloadPlugin(pluginType, name string, version int) *UnloadPlugi
 	return r
 }
 
+// SwapPlugin swaps two plugins with the same type and name e.g. collector:mock:1 with collector:mock:2
+func (c *Client) SwapPlugin(loadPath []string, unloadType, unloadName string, unloadVersion int) *SwapPluginsResult {
+	r := &SwapPluginsResult{}
+
+	if len(loadPath) != 1 {
+		r.Err = errors.New("Did not receive only one plugin to load.")
+		return r
+	}
+	// Check if plugin you are trying to unload is loaded
+	var isLoaded bool
+	lps := c.GetPlugins(false)
```

Comment:
Hey could you elaborate a little? I found
`func (s *Server) getPlugin(w http.ResponseWriter, r *http.Request, p httprouter.Params) ` but not sure if that is what you meant… Thanks